pkg/process/config/yaml_config.go

```diff
@@ -46,6 +46,7 @@ func (a *AgentConfig) loadSysProbeYamlConfig(path string) error {
 
 	a.CollectLocalDNS = config.Datadog.GetBool(key(spNS, "collect_local_dns"))
 	a.CollectDNSStats = config.Datadog.GetBool(key(spNS, "collect_dns_stats"))
+	a.DNSTimeout = config.Datadog.GetInt(key(spNS, "dns_timeout"))
```

Comment:
Should this be configurable?
Also, if this is expecting the timeout in seconds please add a suffix like "_in_s"
(I'd also check if there is a `config.Datadog.GetDuration`)